vehicle-python/src/vehicle_lang/compile/abc.py

```diff
@@ -32,10 +44,19 @@ class Builtins(
         _Nat,
         _Int,
         _Rat,
+        _Variable,
+        _VariableValue,
```

Comment:
We really shouldn't be adding `_VariableValue`. Variable values will always be tensors of `_Nat`, `_Int`, or `_Rat`, and we simply don't have that typing information on the Python side of things, so we should simply treat them as `Vector[Any]`, instead of adding these arguments.